packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -53,6 +53,8 @@ export const makeSelectProps = propsFactory({
     default: '$vuetify.noDataText',
   },
   openOnClear: Boolean,
+  placeholder: String,
+  persistentPlaceholder: Boolean,
```

Comment:
If you remove these from here they'll just be passed through automatically instead.

Reply:
Yes, I removed them